### PR TITLE
fix: docker manifest on draft release

### PR DESCRIPTION
**Pull request by k0da**
docker manifests exepcted to be pushed on draft.

Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

**Comment by k0da** [comment]
waiting on https://github.com/goreleaser/goreleaser/pull/2117

**Comment by caarlos0** [comment]
don't we have a test for that? If not, can you add one?

I do agree it makes sense to push on drafts.

**Comment by k0da** [comment]
@caarlos0 I'll take a look

**Comment by k0da** [comment]
@caarlos0 test is added

File: internal/pipe/docker/docker_test.go [hunk 1]
```diff
@@ -1002,6 +1002,18 @@ func TestDefaultDockerfile(t *testing.T) {
 	require.Equal(t, "Dockerfile", ctx.Config.Dockers[1].Dockerfile)
 }
 
+func TestDraftRelease(t *testing.T) {
+	var ctx = &context.Context{
+		Config: config.Project{
+			Release: config.Release{
+				Draft: true,
+				},
```
Comment (caarlos0) [inline internal/pipe/docker/docker_test.go:1010]: ci will complain about lack of gofmp probably...

Comment (k0da) [reply internal/pipe/docker/docker_test.go]: @caarlos0 thanks! 👍 fixed

**Review by caarlos0 — APPROVED** [review]

**Pull request merged**